### DISCUSSION TITLE
Please add support for new model gemini/gemini-exp-1114 / (also please support exp/all Gemini models on VertexAI)

### DISCUSSION THREAD

**Discussion by Jasmin68k** [cline/cline#740] [Feature Requests]
Hello!

There's a new Gemini model available on https://ai.google.dev/gemini-api/docs/models/experimental-models. Please add.

On Vertex AI https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/gemini-experimental there still is the old experimental Gemini model. They might soon upgrade to the new one as well. In any case, having the experimental Gemini (or actually all Gemini models) from Vertex AI in Cline would also be nice (saves the user from running LiteLLM).

**Comment by RogelioRichmanAstronaut** [comment]
it is added in the Pull Request https://github.com/cline/cline/pull/754, so you can test it

**Comment by elemental-mind** [comment]
Could this be closed as a duplicate of #764 (most votes)?

**Comment by josephomills** [comment]
Exactly. Just like all models on OpenRouter are available in Cline, It would be great to have all Gemini and even other models on Vertex AI added to the list of models. It's still on Claude models at the moment. BTW Gemini 2.0 Flash Exp is also out, so it would be great to have it on there as well.

**Comment by thermite482** [comment]
Hi @elemental-mind 

I believe the request is to add gemini models to the "GCP Vertex AI" API Provider

It's an extension of [this pull request](https://github.com/cline/cline/pull/593)

At the moment, if you select Api Provider as "GCP Vertex AI", only claude models are supported
